crypto/rand/rand_lcl.h

```diff
@@ -17,21 +17,24 @@
 # include <openssl/ec.h>
 # include "internal/rand.h"
 
-/* we require 256 bits of randomness */
-# define RANDOMNESS_NEEDED (256 / 8)
+/* Amount of randomness (in bytes) we want for initial seeding. */
+# define RANDOMNESS_NEEDED              (128 / 8)
```

Comment:
Yes, 256 bits is better than 128 but 128 is enough for most (almost all?) purposes.  It's a trade off.  It is stronger but places a higher burden on the OS's random sources (/dev/random and getrandom will be primary sources).  The number of times `RAND_poll` is called will also come into play.  I did suggest getting more bits for the initial seeding.

I'm not going to object to changing this back to 256.  How about calling `RAND_poll` twice instead of once for initialisation in `ossl_rand.c ` ?